src/Text/Microstache/Type.hs

```diff
@@ -126,3 +128,20 @@ instance Exception MustacheException where
 #else
 instance Exception MustacheException
 #endif
+
+data MustacheWarning
+  = MustacheVariableNotFound Key
+    -- ^ The template contained a variable for which there was no data counterpart in the current context
+  | MustacheDirectlyRenderedValue Key
+    -- ^ A complex value such as an Object or Array was directly rendered into the template
```

Comment:
(Disclaimer: I know sufficiently little about mustache templates that the comment below may not make sense.)

I noticed there are only two possible `MustacheWarnings`, but its counterpart in the `mustache` package ([`SubstitutionError`](http://hackage.haskell.org/package/mustache-2.2.3/docs/Text-Mustache-Render.html#t:SubstitutionError)) has several more constructors. Is there a rationale for which ones are included in `MustacheWarning`?